src/main/java/com/coveo/spillway/storage/AsyncLimitUsageStorage.java

```diff
@@ -78,6 +78,14 @@ public AsyncLimitUsageStorage(LimitUsageStorage wrappedLimitUsageStorage) {
     return wrappedLimitUsageStorage.debugCurrentLimitCounters();
   }
 
+  public void shutdownStorage() {
+    executorService.shutdown();
```

Comment:
Well yes, but it does mean that those events will never get counted. Probably not a problem but still something that could be better easily no?